frappe/patches.txt

```diff
@@ -266,3 +266,4 @@ frappe.patches.v12_0.change_existing_dashboard_chart_filters
 execute:frappe.delete_doc("Test Runner")
 execute:frappe.delete_doc_if_exists('DocType', 'Google Maps Settings')
 execute:frappe.db.set_default('desktop:home_page', 'workspace')
+execute:from frappe.desk.page.setup_wizard.install_fixtures import update_genders;update_genders()
```

Comment:
You do not execute patches like this.
Create a new file in v12_0 folder with this contents
`from frappe.desk.page.setup_wizard.install_fixtures import update_genders;update_genders()`
and then mention the path in here

Reply:
You can actually run patches like this though:

- https://github.com/frappe/frappe/blob/develop/frappe/patches.txt#L67
- https://github.com/frappe/frappe/blob/develop/frappe/patches.txt#L90
- https://github.com/frappe/erpnext/blob/develop/erpnext/patches.txt#L54